google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/StreamWriter.java

```diff
@@ -943,44 +943,8 @@ public void onError(Throwable t) {
         }
         inflightBatch = this.inflightBatches.poll();
       }
```

Comment:
I don't think you need these code any more. You can simply abortInflightRequests(t);